### PR TITLE
feat(page-filters): Phase 1 rollout

### PR DESCRIPTION
**Pull request by davidenwang**
Rolling out page filters to GA for the following pages:

User-feedback:
image

Alerts:
image

Monitors:
image

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
Should we be consistent with monitors and have it on the left?

**Comment by davidenwang** [comment]
> Should we be consistent with monitors and have it on the left?

yup agreed,
https://github.com/getsentry/sentry/pull/32793

**Comment by davidenwang** [comment]
Forgot to include some changes, don't merge yet

File: static/app/components/organizations/pageFilters/utils.tsx [hunk 1]
```diff
@@ -79,8 +79,8 @@ export function isSelectionEqual(selection: PageFilters, other: PageFilters): bo
 export function doesPathHaveNewFilters(pathname: string, organization: Organization) {
   const newFilterPaths = (
     organization.features.includes('selection-filters-v2')
-      ? ['issues', 'user-feedback', 'alerts', 'monitors']
-      : []
+      ? ['issues']
+      : ['user-feedback', 'alerts', 'monitors']
```
[the remark this answers is not in the document]
Comment (davidenwang) [reply static/app/components/organizations/pageFilters/utils.tsx:83]: ahh very good catch, will fix

File: tests/js/spec/views/userFeedback/index.spec.jsx [hunk 2]
```diff
@@ -49,7 +50,12 @@ describe('UserFeedback', function () {
       headers: {Link: pageLinks},
     });
 
-    const wrapper = mountWithTheme(<UserFeedback {...params} />, routerContext);
+    const wrapper = mountWithTheme(
+      <OrganizationContext.Provider value={organization}>
```
Comment (evanpurkhiser) [inline tests/js/spec/views/userFeedback/index.spec.jsx:54]: Hmm, doesn't this get inserted with the routerContext?

**Review by evanpurkhiser — APPROVED** [review]
one comment on the test, but let's gooo

File: tests/js/spec/views/userFeedback/index.spec.jsx (hunk 2 above)
Comment (davidenwang) [reply tests/js/spec/views/userFeedback/index.spec.jsx:54]: with RTL we make a bunch of providers, but not with enzyme

Comment (evanpurkhiser) [reply tests/js/spec/views/userFeedback/index.spec.jsx:54]: yup, you're right

**Pull request merged**